File: Bliki-OldBlog/DebuggerDisplay/DebuggerDisplayCSharp/TestBench/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DebuggerDisplayCSharp;

namespace TestBench
{
	public partial class Form1 : Form
	{
		public ContactTypes ContactType { get; set; }

		[Description("Test123")]
		public Person WhichPerson { get; set; }

		public Form1()
		{
			InitializeComponent();
			ContactType = ContactTypes.Billing;
			WhichPerson = new Person("SomeName", 15);
			var x = ContactType;
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			this.propertyGrid1.SelectedObject = this;
		}
	}
}

File: Bliki-OldBlog/DebuggerDisplay/DebuggerDisplay/TestBench/Form1.vb
Imports DebuggerDisplay

Public Class Form1


    Private _contactType As ContactTypes
    Public Property Contact() As ContactTypes
        Get
            Return _contactType
        End Get
        Set(ByVal value As ContactTypes)
            _contactType = value
        End Set
    End Property


    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Me.PropertyGrid1.SelectedObject = Me
    End Sub
End Class
